docs/extensions/hooks.md

```diff
@@ -239,6 +239,9 @@ export default ({ embed }, { env }) => {
 | `response`                     | `request`, `response`, `ip`, `duration`, `finished` |
 | `auth.login`                   | `payload`, `status`, `user`, `provider`             |
 | `files.upload`                 | `payload`, `key`, `collection`                      |
+| `extensions.loaded`            | `payload`                                           |
+| `extensions.unloaded`          | `payload`                                           |
+| `extensions.reload`            | `payload`, `added` , `removed`                      |
```

Comment:
Sorry for the confusion but our docs are now in a separate repo, but you can just copy these changes into https://github.com/directus/docs and create a separate PR there.